packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4989,11 +4986,11 @@ export class Base64Decode extends Op {
 		assertLen(args.length, 1, line);
 		const argument = args[0];
 		switch (argument) {
-			case "URLEncoding": {
+			case TxFieldEnum.URLEncoding: {
```

Comment:
That's not a Tx field. We should have a separate enum for json encodings

Reply:
Done